client/verta/tests/registry/test_model_dependencies.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+from types import ModuleType
+from typing import Callable, List
+from verta._internal_utils import model_dependencies as md
+
+
+def test_list_class_functions(dependency_testing_model) -> None:
+    """ Verify that all the functions in the test class are recognized and
+    returned.
+    """
+    expected_function_names = [
+        '__init__',
+        'predict',
+        'unwrapped_predict',
+        'make_dataframe',
+        'make_timeout',
+        'make_spacy_error',
+        'make_boto_session',
+        'nested_multiple_returns_hint',
+        'nested_type_hint'
+    ]
```

Comment:
There's probably pros and cons either way, but we might want to have this as an attribute on `DependencyTestingModel`. That way if we ever change the model code, we can update this list right there, instead of having to remember that it's defined in the test code.

Just as a thought—I'm not totally convinced on it, myself.